htmresearch/frameworks/nlp/imbu.py

```diff
@@ -89,22 +89,29 @@ class ImbuModels(object):
   defaultModelType = "CioWordFingerprint"
   defaultRetina = "en_associative"
 
-  def __init__(self, cacheRoot, dataPath, loadPath, savePath,
-      modelSimilarityMetric):
+  # Sequence of classification model types that accept CioEncoder kwargs
+  requiresCIOKwargs = {
```

Comment:
In fluent_all.py to determine whether or not to pass cortical.io credentials, but in retrospect, could be used in _model factory() here, too.  Will follow up tomorrow when I have my computer.